### ISSUE TITLE
XLSX support?

### ISSUE THREAD

**Issue by mindfuldan** [jecovier/vue-json-excel#19]
Apologies if this has been addressed already first and foremost.

I really like this component that you have made and am wondering if there is an ability to support changing the extension to xlsx in the near future?

ty so much :)

**Comment by jecovier** [comment]
hi @mindfuldan, thanks for your words. Sadly I don't have the time to implement that feature at this moment, but I hope that can implement in the future. If you are interesting in something like that, check this library, I think it's a good start: https://github.com/SheetJS/js-xlsx

**Comment by mindfuldan** [comment]
ty for the response sir!!

I am curious with the library that you suggested if you were to implement
this feature would this be what you would use to look to get this to work??

**Comment by jecovier** [comment]
@mindfuldan yes, I think this would be a potential solution for the xlsx format.

**Comment by juanbrujo** [comment]
this worked for me https://lovemewithoutall.github.io/it/json-to-excel/
